tracpro/baseline/forms.py

```diff
@@ -32,7 +33,8 @@ def __init__(self, *args, **kwargs):
         super(BaselineTermForm, self).__init__(*args, **kwargs)
 
         if org:
-            polls = Poll.objects.active().by_org(org).order_by('name')
+            polls = Poll.objects.active().by_org(org)\
+                .annotate(lcase_name=Lower('name')).order_by('lcase_name')
```

Comment:
Can this be simplified as 
`polls = Poll.objects.active().by_org(org).order_by(Lower('name'))`

I don't think annotate is necessary here